pkg/kubelet/network/dns/dns.go

```diff
@@ -84,7 +85,7 @@ func NewConfigurer(recorder record.EventRecorder, nodeRef *v1.ObjectReference, n
 	}
 }
 
-func (c *Configurer) getClusterDNS(pod *v1.Pod) []net.IP {
+func (c *Configurer) getClusterDNS(pod *v1.Pod) ([]net.IP, error) {
 	// todo: consider using exported const of default network name from arktos package
 	const defaultNetwork = "default"
 	networkName, ok := pod.Labels["arktos.futurewei.com/network"]
```

Comment:
seems we have a const for "arktos.futurewei.com/network"

Reply:
yes; will reference the const.